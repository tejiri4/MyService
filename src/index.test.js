import Index from "./";

describe("Index file", () => {
  it("renders the index file without crashing", () => {
    expect(
      JSON.stringify(
        Object.assign({}, Index, { _reactInternalInstance: "censored" }),
      ),
    ).toMatchSnapshot();
  });
});
