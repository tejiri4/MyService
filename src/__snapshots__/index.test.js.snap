// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index file renders the index file without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
